script/bulk_import/vbulletin.rb

```diff
@@ -13,10 +51,19 @@ def initialize
     username = ENV["DB_USERNAME"] || "root"
     password = ENV["DB_PASSWORD"]
     database = ENV["DB_NAME"] || "vbulletin"
+    charset  = ENV["DB_CHARSET"] || "utf8"
```

Comment:
Getting automatically is not correct in some cases. I mean, the database, even the table could be `utf8_general_ci`, but the data that the vBulletin saved still in `latin1`.